frontend/src/routes/FacilityBooking/FacilityLandingPage/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { useHistory } from 'react-router-dom'
+import styled from 'styled-components'
+import { PATHS } from '../../Routes'
+import ButtonComponent from '../../../components/Button'
+import BottomNavBar from '../../../components/Mobile/BottomNavBar'
+import TopNavBar from '../../../components/Mobile/TopNavBar'
+
+export default function FacilityLandingPage({ onLeftClick }: { onLeftClick?: () => void }) {
+  const history = useHistory()
+  const TopBar = styled.div`
+    position: -webkit-sticky;
+    position: sticky;
+    top: 0;
+    width: 100%;
+    height: 50px;
+    background-color: #de5f4c;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 10px;
+    border-radius: 0px 0px 10px 10px;
+    z-index: 999;
+  `
+  const SearchContainer = styled.div`
+    display: flex;
+    justify-content: space-between;
+    height: 154px;
+    width: 297px;
+  `
+  const LandingPageContainer = styled.div`
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-direction: column;
+    width: 297px;
+    height: 234px;
+    margin: 63px 35px 0px 43px;
+  `
+  return (
+    <>
+      <TopNavBar title="Faclility Booking" />
+      <LandingPageContainer>
+        <SearchContainer>
+          <ButtonComponent
+            state="primary"
+            text="Search by facility"
+            width="129px"
+            height="104px"
+            fontSize="20px"
+            size="small"
+            onClick={() => {
+              history.push(PATHS.FACILITY_BOOKING_MAIN)
+            }}
+          />
+          <ButtonComponent
+            state="primary"
+            text="Search by Date/Time"
+            width="129px"
+            height="104px"
+            fontSize="20px"
+            size="small"
+            onClick={() => {
+              history.push(PATHS.FACILITY_BOOKING_MAIN)
+            }}
```

Comment:
```suggestion
            onClick={() => {
              // TODO
            }}
```

Reply:
for now we just comment this out ya since we haven't implement the search by date/time yet